src/components/DeckMechanisms/index.js

```diff
@@ -435,24 +457,7 @@ export default class DeckMechanisms extends React.Component {
   }
 
   reset = () => {
-    this.setState(
-      {
-        hand: [],
-        RNG: 'REGULAR',
-        hasCycledThisTurn: false,
-        specifics: {
-          activeFrozenCores: 0,
-          liveDawnsparks: false,
-          noUnitsOnFirstTurn: true,
-          potentialFrozenEnemies: false,
-        },
-        turn: this.props.turn,
-        mana: DEFAULT_MANA + (this.props.turn - 1),
-        deck: resolveDeckWeight(this.props.deck),
-        playerOrder: 'FIRST',
-      },
-      this.drawHand
-    )
+    this.setState(this.DEFAULT_STATE, this.drawHand)
```

Comment:
```suggestion
    this.setState({ ...DEFAULT_STATE }, this.drawHand)
```

Reply:
I don't understand the difference. Is it that it will create a copy instead of just reassigning it?

Reply:
Yes. :)